osbenchmark/worker_coordinator/worker_coordinator.py

```diff
@@ -1562,17 +1576,32 @@ async def __call__(self, *args, **kwargs):
         finally:
             yappi.stop()
             s = python_io.StringIO()
-            yappi.get_func_stats().print_all(out=s, columns={
+
+            stats = yappi.get_func_stats()
+
+            if self.sort_type:
```

Comment:
Enforce this check while accepting the params.
e.g. 
```
parser.add_argument(
    '--mode',
    type=str,
    choices=['normal', 'advanced', 'expert'],  # List of valid options
    required=True,  # Make this argument mandatory
    help="Choose a mode: 'normal', 'advanced', or 'expert'"
)
```
This way argparser will throw exception.